crypto/x509v3/v3_alt.c

```diff
@@ -303,10 +303,12 @@ static int copy_email(X509V3_CTX *ctx, GENERAL_NAMES *gens, int move_p)
     ASN1_IA5STRING *email = NULL;
     X509_NAME_ENTRY *ne;
     GENERAL_NAME *gen = NULL;
-    int i;
+    int i = -1, email_nid = NID_pkcs9_emailAddress;
```

Comment:
why have a local variable that has a value of a constant, just to make an indent loop "prettier"  ugh.
if you really want to do this, mail email_nid be static const.